gratipay/models/package/__init__.py

```diff
@@ -40,3 +51,40 @@ def from_names(cls, package_manager, name):
         """
         return cls.db.one("SELECT packages.*::packages FROM packages "
                           "WHERE package_manager=%s and name=%s", (package_manager, name))
+
+
+    @property
+    def team(self):
+        """A computed attribute, the :py:class:`~gratipay.models.team.Team`
+        linked to this package if there is one, otherwise ``None``. Makes a
+        database call.
+        """
+        return self.load_team(self.db)
```

Comment:
`self.db` exposes the same `run`, `one`, `all` API as a cursor does, but with less control over the transaction. [Docs](http://postgres-py.readthedocs.io/en/latest/#context-managers).